packages/scss/src/components/index.scss

```diff
@@ -44,7 +44,7 @@
 @forward 'button'; // 12 Ko
 @forward 'textfield'; // 13 Ko
 @forward 'util'; // 14 Ko
-@forward 'navSide'; // 15 Ko
+@forward 'navside'; // 15 Ko
```

Comment:
Sure about this? 🤔

Reply:
The actual file is lowercase. This line of code was breaking the build on my WSL setup.